web-client/src/views/DocketEntryQc.tsx

```diff
@@ -125,7 +124,6 @@ export const DocketEntryQc = connect(
             onConfirmSequence={completeDocketEntryQCAndSendMessageSequence}
           />
         )}
-        {showModal === 'FileUploadStatusModal' && <FileUploadStatusModal />}
```

Comment:
Is this being removed because it isn't pertinent to Docket Entry QC? (Sorry if this is an obvious question!)

Reply:
Cool, thank you!